src/elements/Step/StepContent.js

```diff
@@ -57,7 +57,11 @@ StepContent.propTypes = {
   /** Primary content of the StepTitle. Mutually exclusive with children. */
   title: customPropTypes.every([
     customPropTypes.disallow(['children']),
-    PropTypes.node,
+    PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.number,
+      PropTypes.object,
+    ])
```

Comment:
Yep, I'm just making separate PRs from #449 since it got so large.  Also, the customPropType for shorthand needs more thought, per the the discussion in #449.  Not sure what the final PR will look like for that, but taking it one step at a time.